### PR TITLE
- fix battery cycles not showing up in the popup.

### PR DESCRIPTION
**Pull request by Military-Doctor**
When I review the code, I found that battery cycles are not showing up in the popup.

So I added a field `cycles` and place it at the bottom.

image

### CODE REVIEW

File: Modules/Battery/popup.swift [hunk 1]
```diff
@@ -139,12 +140,14 @@ internal class Popup: NSView, Popup_p {
         let separator = SeparatorView(LocalizedString("Details"), origin: NSPoint(x: 0, y: self.detailsHeight-Constants.Popup.separatorHeight), width: self.frame.width)
         let container: NSView = NSView(frame: NSRect(x: 0, y: 0, width: self.frame.width, height: separator.frame.origin.y))
         
-        self.levelField = PopupRow(container, n: 3, title: "\(LocalizedString("Level")):", value: "")
-        self.sourceField = PopupRow(container, n: 2, title: "\(LocalizedString("Source")):", value: "")
-        let t = self.labelValue(container, n: 1, title: "\(LocalizedString("Time")):", value: "")
+
```
Comment (exelban) [inline Modules/Battery/popup.swift:143]: Unnecessary line

**Comment by exelban** [comment]
Hi. Thanks for your contribution.

**Comment by exelban** [comment]
Resolve the merge conflict, please.

**Comment by Military-Doctor** [comment]
Resolved.

**Pull request closed without merging**